cmd/machine-config-daemon/pivot.go

```diff
@@ -38,57 +35,15 @@ func init() {
 	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
 }
 
-func run(_ *cobra.Command, args []string) (retErr error) {
-	flag.Set("logtostderr", "true")
-	flag.Parse()
-
-	var container string
-	if fromEtcPullSpec || len(args) == 0 {
-		fromEtcPullSpec = true
-		data, err := os.ReadFile(etcPivotFile)
```

Comment:
Hmm, I just realized we have a comment that says:
```
	// etcPivotFile is used for 4.1 bootimages and is how the MCD
	// currently communicated with this service.
```
on this file. I... don't think that should be the case anymore? If you installed on 4.1, upgraded to 4.14, then scaled up, I don't think it touches this codepath at all.

If we want to be EXTRA safe maybe we can consider testing that, but I am pretty confident the comment should be outdated

Reply:
Understood, I've taken off all references/calls to `etcPivortFile` it in the pivot.go file. Our verify test complained too, said the var was never used, which makes sense. Let's see what the tests say, but I agree that this should be OK.